lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_git_helper.rb

```diff
@@ -56,17 +53,27 @@ def self.localize_project
         #       in the release-toolkit instead, and move this code away from `ios_git_helper`.
         #
         def self.update_metadata
-          Action.sh("cd #{ENV['PROJECT_ROOT_FOLDER']} && ./Scripts/update-translations.rb")
+          Action.sh("cd #{get_from_env!(key: 'PROJECT_ROOT_FOLDER')} && ./Scripts/update-translations.rb")
```

Comment:
Can't wait to get rid of the python script, of the odd requirement of having to `cd` to the right dir first for the script to work as expected, and for that script to be hosted in the app repo… 

![soon meme](https://i.kym-cdn.com/entries/icons/original/000/005/673/sooon.jpg)